### ISSUE TITLE
Cant pick up events from private channels 

### ISSUE THREAD

**Issue by hallowcard13** [nuxt-community/laravel-echo-module#67]
```
 this.$echo.private('PrivateChannel')
            .listen('.custom-event', (e) => {
                console.log('e');
            });
```

**Comment by devzom** [comment]
Is the backend working correctly?
Look into this [issue #9](https://github.com/nuxt-community/laravel-echo-module/issues/9)

mostly there's an issue with channelName or eventName but first check the Redis config from above link

**Comment by hallowcard13** [comment]
> Is the backend working correctly? Look into this [issue #9](https://github.com/nuxt-community/laravel-echo-module/issues/9)
> 
> mostly there's an issue with channelName or eventName but first check the Redis config from above link

Yes its, actually public channels are working perfectly fine, BUT FOR private channels it's not

**Comment by devzom** [comment]
Alright, is move deeper. Is the Auth correct?
 Are You receiving in redis logs that user is authenticated?

**Comment by hallowcard13** [comment]
> Alright, is move deeper. Is the Auth correct? Are You receiving in redis logs that user is authenticated?

Well im not using redis, i'm using pusher, and yes auth is working fine. I can connect to websocket but i cant see any subscription to the private channel

**Comment by hallowcard13** [comment]
Client is being connected and authenticated properly but checking the logs its not subscribing to the private  channel.
<img width="1792" alt="Screen Shot 2022-02-03 at 9 15 53 PM" src="https://user-images.githubusercontent.com/4696933/152349979-af3755d9-0d02-4267-97a3-395a1ec53991.png">

<img width="1215" alt="Screen Shot 2022-02-03 at 9 14 13 PM" src="https://user-images.githubusercontent.com/4696933/152349724-9f8bbb4c-fd8f-4b15-86f1-9a589ea53420.png">

**Comment by devzom** [comment]
Alright, self-hosted pusher?
I see You're connected and AUTH gave [200].

There's an issue with NPM echo packages, which version are You using, show me the part of package.json.

#TIP 
You should use the DevTools > Network > WS tab to inspect Websocket related stuff.
![Zrzut ekranu 2022-02-14 o 11 05 02](https://user-images.githubusercontent.com/1417473/153843724-fbdccbcb-f4a4-4772-81e5-479b51c12ca3.png)
